ci/L0_multi_gpu/test.sh

```diff
@@ -42,7 +42,7 @@ rm -rf models && mkdir -p models
 cp -r ${SAMPLE_MODELS_REPO}/vllm_model models/vllm_opt
 sed -i '3s/^/    "tensor_parallel_size": 2,\n/' models/vllm_opt/1/model.json
 
-pip3 install tritonclient grpcio nvidia-ml-py3
+python3 -m pip install --upgrade pip && pip3 install tritonclient[all] nvidia-ml-py3
```

Comment:
Do you need both clients or just the grpc client in `tritonclient[all]`?

Reply:
Thank you for pointing this out. We need only GPC. Modified it to `tritonclient[grpc]`.